content/en/docs/reference/faq.md

```diff
@@ -0,0 +1,231 @@
+
+  > Where "serving-sr5v2-ksvc-sbtgr.default.openfunction.dev" is the URL address obtained from the command "kubectl get ksvc".
+
+  ```shell
+  10.233.47.71 serving-sr5v2-ksvc-sbtgr.default.openfunction.dev
+  ```
+
+After the above configuration is done, you can get the URL address of the function with the following command. Then you can trigger the function via `curl` or your browser.
+
+```shell
+$ kubectl get ksvc
+
+NAME                       URL
+serving-sr5v2-ksvc-sbtgr   http://serving-sr5v2-ksvc-sbtgr.default.openfunction.dev
+```
+
+## Q: How to enable and configure concurrency for functions?
+
+A: OpenFunction categorizes function types into "[sync runtime](../../concepts/function/#the-sync-runtime)" and "[async runtime](../../concepts/function/#the-async-runtime)" based on the type of request being handled. These two types of functions are driven by Knative Serving and Dapr + KEDA.
+
+Therefore, to enable and configure the concurrency of functions, you need to refer to the specific implementation in the above components. 
+
+The following section describes how to enable and configure concurrency of functions in OpenFunction according to the "[sync runtime](../../concepts/function/#the-sync-runtime)" and "[async runtime](../../concepts/function/#the-async-runtime)" sections.
+
+### Sync runtime
+
+You can start by referring to this [document](https://knative.dev/docs/serving/autoscaling/concurrency/) in Knative Serving on enabling and configuring concurrency capabilities.
+
+Knative Serving has **Soft limit** and **Hard limit** configurations for the concurrency feature.
+
+#### Soft limit
+
+You can refer to the `Global(ConfigMap)` and `Global(Operator)` sections of this [document](https://knative.dev/docs/serving/autoscaling/concurrency/#soft-limit) to configure global concurrency capabilities.
+
+And for `Per Revision` you can configure it like [this](../../concepts/function_scaling_trigger/).
+
+```yaml
+apiVersion: core.openfunction.io/v1beta1
+kind: Function
+metadata:
+  name: function-sample
+spec:
+  serving:
+    scaleOptions:
+      knative:
+        autoscaling:
+          target: "200"
+```
+
+#### Hard limit
+
+OpenFunction currently doesn't support configuring Hard limit for `Per Revision`. You can refer to the `Global(ConfigMap)` and `Global(Operator)` sections of this [document](https://knative.dev/docs/serving/autoscaling/concurrency/#hard-limit) to configure global concurrency capabilities.
+
+This means you can configure Knative Serving's Autoscaling-related configuration items for each function as follows, just as long as they can be passed in as ***Annotation***, otherwise you can only do global settings.
+
+```yaml
+# Configuration in Knative Serving
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-go
+  namespace: default
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/<key>: "value"
```

Comment:
The hard limit is not using annotation, it uses a field in spec:
![image](https://user-images.githubusercontent.com/18525465/174796282-b9ce729a-3879-4d15-9820-33ec3f060c03.png)
https://knative.dev/docs/serving/autoscaling/concurrency/#hard-limit

Reply:
ok